Replace one of our bad exercises with a better one ;P (#127)
rubymonstas-zurich/rubymonstas-zurich.github.io: exercises/boolean-operators.md

    You can tell what the result will be without knowing the value of `x`. Why?

3.  Imagine you are developing an application that controls a cat door. There are 3 cats that you need to accommodate, Alpha, Bravo and Charlie.

>     a) Alpha is hurt and therefore shouldn't leave the house at the moment. Complete the following code to account for this:

Reviewer: ```suggestion
    a) Alpha is hurt and therefore shouldn't leave the house at the moment. Complete the following code so that Alpha can't leave the house:
```

    ```ruby
    def can_alpha_open_door?
      # your code goes here
    end
    ```

    b) Bravo likes to run away and sometimes does not come back for days if he is out between 20:00 and 06:00. Therefore, we want to lock the door to him if it's this late. Otherwise he can leave the house.

    ```ruby
    def can_bravo_open_door?(hour_of_day)
      # your code goes here
    end
    ```

>     c) Charlie is a bit more complicated. Charlie should only be allowed out if both Alpha is inside (since they don't get along), and if Bravo is already inside (because they like to chase each other). Complete the following code to reflect these rules:

Reviewer: ```suggestion
    c) Charlie is a bit more complicated. Charlie should only be allowed out if both Alpha is inside (since they don't get along), and if Bravo is already inside (because they like to chase each other). Complete the following code to let Charlie out just if the two other cats are already inside:
```
